src/modules/Plans/__tests__/PlanRow.test.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import withQueryClient from '_/components/QueryClientHoc';
+
+import { IMustGatherContext, MustGatherContext } from '@app/common/context';
+import { cleanup, render } from '@testing-library/react';
+
+import { FlatPlan } from '../data';
+import PlanRow from '../PlanRow';
+import { fieldsMetadata } from '../PlansPage';
+
+import MERGED_MOCK_DATA from './mergedMockData.json';
+
+afterEach(cleanup);
+
+describe('Plan rows', () => {
+  const planTuples = MERGED_MOCK_DATA.map((plan) => [plan.name, plan]);
+  test.each(planTuples)('%s', (description, plan) => {
+    const Wrapped = withQueryClient(() => (
+      <MemoryRouter>
+        <MustGatherContext.Provider
+          value={
+            {
+              withNs: jest.fn(),
+              latestAssociatedMustGather: jest.fn(),
+            } as unknown as IMustGatherContext
```

Comment:
That's an interesting typing assertion.  Does it need a double casting?

Reply:
yep. This is what the linter suggested. The problem is that this interaface contains multiple methods. For the purpose of this test we are mocking only those that are actually used by our code.